web/arxiv_info.js

```
@@ -173,30 +546,1367 @@ function initializeArxivInfo(pdfDocument) {
+              $(`#${popupId} .toggle-button`).removeClass("active");
+            });
+          } finally {
+            isFetching = false;
+          }
+        });
+
+        // Function to call Claude API
+        async function callClaudeAPI(paperDetails) {
+          // Store the API key in a variable - you should replace this with your actual API key
+          // Consider using environment variables or a secure method to store API keys in production
+          const anthropicApiKey = "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"; // Replace with actual key
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)